helm/Chart.yaml

```diff
@@ -1,15 +1,17 @@
 apiVersion: v1
-name: ack-sfn-controller
-description: A Helm chart for the ACK service controller for sfn
-version: v0.0.2
-appVersion: v0.0.2
-home: https://github.com/aws/aws-controllers-k8s
+name: sfn-chart
+description: A Helm chart for the ACK service controller for AWS Step Functions (Step-Functions)
+version: v0.0.0-non-release-version
+appVersion: v0.0.0-non-release-version
```

Comment:
Hrmmm. Either you didn't do `git fetch --all --tags` or there is no Git tag for v0.0.2 (more worrying)...

Reply:
There are no tags on this repo. https://github.com/aws-controllers-k8s/sfn-controller/tags
I will use env variable to set this to a released version.